packages/contracts/test/src/vault/MaturityVaultTest.t.sol

```diff
@@ -25,14 +25,16 @@ contract MaturityVaultTest is Test {
     uint256 private constant INITIAL_BALANCE = 1e6;
 
     function setUp() public {
-        helperConfig = new HelperConfig(true);
-        params = new ParamsFactory(helperConfig.getNetworkConfig()).createMaturityVaultParams();
+        deployer = new TestDeployVaultsSupport();
+        (ERC20 cbl, ERC20 usdc,) = deployer.deploy();
```

Comment:
Invokes the `deploy()` function to deploy the Vaults Support contracts, configured by the `test.toml`.